src/std/build-spec.ss

```diff
@@ -240,9 +240,6 @@
     "net/smtp/headers"
     "net/smtp/api"
     "net/smtp"
-    "net/websocket/server"
```

Comment:
Why this?

Reply:
This would make the websocket server un-importable, no?

Reply:
it was duplicate, it is repeared right underneath.